extra_data/__init__.py

```diff
@@ -39,6 +39,8 @@
 from .reader import *
 from .stacking import *
 from .utils import *
+from .filecache import get_global_filecache
```

Comment:
I don't think this needs to be exposed at the top level of extra_data - it's a detail that won't matter for most users.

Reply:
Referring to a previous comment, users have to use it. But they need to close files relatively rare. So, let it be exposed at `extra_data.filecache`.